Automattic/woocommerce-payments
# Added docs for cancel_authorization endpoint.

Fixes https://github.com/Automattic/woocommerce-payments-meta/issues/252

#### Changes proposed in this Pull Request

Added cancel authorization endpoint to the docs and resolved small issue with the status.


#### Testing instructions

* Make sure that you can see cancel authorization API docs in the documentation

-------------------


**Post merge**
